mozmeao/birdbox
# Add a horizontal image block which can display an SVG (eg above the hero)

## Description

This changeset adds a new StreamField block that can be used to insert an SVG image into the page, on a colored background panel. It can sit above a hero - eg here's an example with the defalt spacing around it.

Note that the width/ratio of the image will have an impact on how well it 'sits' on the page. I've provided some protocol-compliant spacing options around the image, but we don't have free range over it via the CMS

<img width="1640" alt="Screenshot 2024-05-28 at 15 35 25" src="https://github.com/mozmeao/birdbox/assets/101457/73c06df9-6dd2-48ba-a545-d08215976de5">

<img width="920" alt="Screenshot 2024-05-28 at 16 44 43" src="https://github.com/mozmeao/birdbox/assets/101457/d0377e26-9ef6-4b45-95cf-baec7fd4aa87">



- [X] I have manually tested this.

### Issue

Resolves #372

### Testing

* Add an SVG image to the Image library
* Add the Horizontal Image / Wordmark block to a streamfield and select the SVG image
* Save/preview/publish as usual
* Edit the page again and tweak the spacing options to see how they are
